vulnfeeds/cmd/combine-to-osv/main.go

```diff
@@ -52,23 +52,26 @@ func loadInnerParts(innerPartInputPath string, output map[string][]vulns.Package
 		Logger.Fatalf("Failed to read dir? %s", err)
 	}
 	for _, entryInner := range dirInner {
+		if !(strings.HasSuffix(entryInner.Name(), ".json")) {
```

Comment:
nit: the other ( ) are not necessary.l

Reply:
Fixed in https://github.com/google/osv.dev/pull/1535/commits/a3d83b778c926100bf37bbdee37133565236db83